routes/campaigns/campaign.php

```diff
@@ -47,6 +47,9 @@
 Route::get('/w/{campaign}/recovery', 'Campaign\RecoveryController@index')->name('recovery');
 Route::post('/w/{campaign}/recovery', 'Campaign\RecoveryController@recover')->name('recovery.save');
 
+Route::get('/w/{campaign}/recovery-posts', 'Campaign\RecoveryController@posts')->name('recovery.posts');
```

Comment:
Please make the route recovery/posts and the controller `Campaign\Recovery\PostController@index`